### ISSUE TITLE
[0.3.0 QA] In BTree Index Benchmark, some cases(Q1, Q2) oap-with-index is slower than the oap-without-index

### ISSUE THREAD

**Issue by carsonwang** [Intel-bigdata/OAP#518]
Reporter: Hao
In BTree Index Benchmark, I saw some cases(Q1, Q2) oap-with-index is slower than the oap-without-index, the index statistic issues?

**Comment by Moonlit-Sailor** [comment]
Q1 is full table scan.
Test Q2, results are shown below:
`SELECT * FROM store_sales WHERE ss_ticket_number < 2000000`

+--------------------+------+------+-----+------+-----+-----+-------+
 |　　 　Q2　　　　|T1/ms|T2/ms|T3/ms|T4/ms|  T5　| T6　 |Avg/ms|
+--------------------+------+------+-----+------+-----+-----+-------+
 |oap-without-index |22715|17475  |17539|16520 | 9117| 7528| 15149    |
 |　oap-with-index　|21851|17752 |16843|15501| 8833| 7835| 14769    |
+--------------------+------+------+-----+------+-----+-----+-------+


3 data files are involved in this query.
In oap-without-index test, these 3 data files are full file scanned to fetch all of its rows.
While in oap-with-index test, two files are using full file scan to fetch all of its rows, one file is using index scan, which is much faster than the other two files.
In conclusion, due to the two files full scan, the performance of oap-with-index is similar with that of oap-without-index.

**Comment by carsonwang** [comment]
Thanks @Moonlit-Sailor for the report. Closing this.
